### ISSUE TITLE
SEMGREP_URL not reflected in log message

### ISSUE THREAD

**Issue by snyamathi** [semgrep/semgrep#5753]
**Describe the bug**
When `SEMGREP_URL` is set, the logged message `Fetching rules from https://semgrep.dev/registry` doesn't use the `SEMGREP_URL` which is where the rules are _actually_ fetched from

**To Reproduce**
```
$ SEMGREP_URL=http://localhost semgrep scan -c p/foo bar.js
Fetching rules from https://semgrep.dev/registry.
```

**Expected behavior**
```
$ SEMGREP_URL=http://localhost semgrep scan -c p/foo bar.js
Fetching rules from http://localhost
```

**What is the priority of the bug to you?**

- [ ] P0: blocking your adoption of Semgrep or workflow
- [ ] P1: important to fix or quite annoying
- [x] P2: regular bug that should get fixed

**Environment**
an official binary, v0.104.0

**Use case**
What will fixing this bug enable for you?

Peace of mind that I'm pulling from the correct data source

**Comment by r2c-demo** [comment]
This issue is synced in Linear at https://linear.app/r2c/issue/PA-1653/semgrep-url-not-reflected-in-log-message. _Note_: this link is for r2c use only and is not accessible publicly.

**Comment by snyamathi** [comment]
Took a quick stab at this: https://github.com/returntocorp/semgrep/pull/5754

**Comment by ievans** [comment]
Closed in #5754
